src/util/StringStream.js

```diff
@@ -73,9 +74,14 @@ class StringStream {
   current(){return this.string.slice(this.start, this.pos)}
   hideFirstChars(n, inner) {
```

Comment:
Ah, I see, I expanded the diff, but apparently not far enough, sorry. Since string stream objects are being created by the thousands during editing, adding an extra field is a non-trivial cost. Could you take a rough look at the pug mode to see if just changing the behavior of `hideFirstChars` would cause any damage there? I'd prefer to simply do that, and not introduce a new method.